muikku-core-plugins/src/main/resources/META-INF/resources/scripts/src/sass/elements/ordered-container.scss

```diff
@@ -17,7 +17,7 @@
 
 .ordered-container__error-title {
   align-items: center;
-  background-color: $color-panel-background;
+  background-color: $color-panel-background-color;
```

Comment:
Ordered container still in use? @jjlankinen ?

Reply:
Unfortunately yes, this will be merged to devel and ordered-container is still present. It will be removed later on but currently it's unnecessary complicated to follow the material-redesign changes. We need to check these later of course.